backend/battling/api/endpoints.py

```diff
@@ -1,11 +1,21 @@
 from django.db.models import Q
 
 from rest_framework import generics, permissions
+from rest_framework.response import Response
+from rest_framework.views import APIView
 
-from battling.api.serializers import BattleSerializer, CreateTeamSerializer
+from battling.api.serializers import BattleSerializer, CreateTeamSerializer, UserSerializer
 from battling.models import Battle, Team
 
 
+class LoggedUser(APIView):
+    permission_classes = [permissions.IsAuthenticated]
+
+    def get(self, request):
+        serializer = UserSerializer(request.user, context={"request": request})
```

Comment:
hmmm, why do you need the request?

Reply:
Because when I display the detail battle page, I need to know who is logged, so only users who are the creator or the opponent can see the page